test/strings.jl

```diff
@@ -1719,3 +1719,102 @@ d = UTF32String(c)
 c[1] = 'A'
 @test d=="A"
 
+# Test invalid sequences
+
+byt = 0x0
```

Comment:
is this line needed? and unnecessary indentation of the chunk of tests below this